client/src/Routes.js

```diff
@@ -81,7 +92,7 @@ const routes = {
 function getAvailableNavbarLinks(loggedUser) {
   switch (loggedUser?.role) {
     case UserRoles.CLIENT:
-      return [userLogoutRouteName];
+      return [userLogoutRouteName, clientDetailsRouteName];
```

Comment:
I think that the logged client should access its details by clicking in the user icon top right.

The best way is to add in the authState (AuthContext) a field `userIconLinkPath` that will contain the path to navigate to, when the user icon is clicked. 
The path is chosen by assessing the user role (switch on the currentUser role => if client userIconLinkPath:"/client").

Then, the userIconLinkPath is recovered by the authContext and passed by the pages to the navbar that associates it to the icon.